### ISSUE TITLE
The bug of ShapeLLM about the grounding answer.

### ISSUE THREAD

**Issue by 1768637871** [qizekun/ShapeLLM#4]
When I use the command
`python -m llava.serve.cli --model-path qizekun/ShapeLLM_13B_gapartnet_v1.0 --pts-file assets/instrument.npy`
And I always get the answer of hinge door.
<img width="640" alt="image" src="https://github.com/qizekun/ShapeLLM/assets/74090298/f729b347-a1e0-4ed4-a6c3-cbf3da3fd670">
But the cloud in instrument.npy is not a hinge door. The Recon++ I used is zeroshot/large/best_lvis.pth.

**Comment by qizekun** [comment]
Hi,

For common general understanding tasks, such as VQA, we employ the objaverse dataset for supervised finetuning. You can test this using the `gizekun/ShapeLLM_13B_general_v1.0` model. For Grounding tasks, we train using the GAPartNet dataset, and you may select samples from the GAPartNet test set for evaluation.
